graph/topological_sort.ts

```diff
@@ -0,0 +1,49 @@
+export function createGraph(v: number): number[][] {
```

Comment:
These utilities don't belong here. If they're handy in the tests, define them there. Long-term, an adjacency-list based graph class is probably the ideal solution.